.github/workflows/rosa-cluster-create.yml

```
@@ -3,13 +3,16 @@ name: ROSA Cluster - Create
 on:
   workflow_dispatch:
     inputs:
+      clusterName:
+        description: 'Name of the cluster'
+        type: text
```

I think you can use:

```suggestion
        type: text
        default: gh-${{ github.repository_owner }}
```

This makes it a little bit more readable below as you don't need the `Determine cluster name` step + you can preview the value when running the workflow.